src/parser/vast_parser.js

```diff
@@ -114,7 +114,7 @@ export class VASTParser extends EventEmitter {
       this.emit('VAST-resolving', { url, wrapperDepth, originalUrl });
 
       this.urlHandler.get(url, this.fetchingOptions, (err, xml) => {
-        this.emit('VAST-resolved', { url, error: err });
+        this.emit('VAST-resolved', { url, error: err, xml, wrapperDepth });
```

Comment:
Sorry that context is not enough.
`wrapperDepth` is not used in client now.
But, It may be used in later.
cc. https://tapjoy.slack.com/archives/C0117V6DFUP/p1659402512899889